src/app/patient-dashboard/oncology/program-snapshot/oncology-program-snapshot.component.ts

```diff
@@ -30,7 +32,12 @@ export class OncologyProgramSnapshotComponent implements OnInit, OnDestroy {
         this.hasError = true;
       } else {
         this.hasData = false;
-        this.loadOncologyDataSummary(patientUuid);
+        if (this.programUuid === '37ff4124-91fd-49e6-8261-057ccfb4fcd0') {
+          this.loadScreeningAndDiagnosisData(patientUuid);
```

Comment:
It seems `this.loadScreeningAndDiagnosisData` loads the recent data for that patient enrolled in screening and diagnosis program. What happens to the historical data for the same patient if he/she was previously enrolled in either breast cancer screening or lung cancer screening?

Reply:
@corneliouzbett The function queries the db for specific encounters which have been saved. So for all the screenings done the etl endpoint will return payloads for respective screening encounters.
https://github.com/AMPATH/etl-rest-server/pull/818

Reply:
I see  👍🏽